docs/pages/reference/cli.mdx

```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
 h1: Command Line (CLI) Reference
 ---
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): A process and CLI tool that supports the Teleport access plane.
+- [tsh](#tsh): A CLI tool that allows end users to interact with Teleport sessions and list cluster information.
+- [tctl](#tctl): An administrative CLI tool that can configure the Teleport Auth Service.
+- [tbot](#tbot): A CLI tool that supports Machine ID on Teleport, used to apply short lived certificates to service accounts (e.g, a CI/CD server).
 
 (!docs/pages/includes/permission-warning.mdx!)
 
 (!docs/pages/includes/backup-warning.mdx!)
 
 ## teleport
 
-The Teleport daemon is called `teleport`. It can be configured to run one or
-more "roles" with the `--roles` flags. The arguments to `--roles` correspond to
-the following services.
-
-<Notice type="tip" scope={["cloud"]}>
-
-Teleport Cloud manages Teleport instances with the `auth` and `proxy` roles. Use
-the remaining roles to manage access to specific resources and other Teleport
-clusters.
-
+The CLI tool that supports the Teleport access plane is called `teleport`, and allows Teleport services to be created, configured, and otherwise interacted with over the command line.
```

Comment:
- Same comment as above re: "supports the Teleport access plane"
- What does it mean for a Teleport service to be "created"? Should we say instead that it runs Teleport services, and list the services that it runs?